ProcessMaker/Repositories/TokenRepository.php

```diff
@@ -145,6 +145,8 @@ public function persistActivityActivated(ActivityInterface $activity, TokenInter
             }
         }
 
+        $this->getAcionByEmail($activity);
```

Comment:
This method does not return nothing and does not set any variable, is this correct?